documentation/resources/adoptopenjdk_install.md

```diff
@@ -0,0 +1,45 @@
+[back to resource list](https://github.com/sous-chefs/java#resources)
+
+# adoptopenjdk_install
+
+Introduced: v7.0.0
+
+## Actions
+
+- `:install`
+- `:remove`
+
+## Properties
+
+| Name                  | Type            | Default                                  | Description                                         | Allowed Values                         |
+| --------------------- | --------------- | ---------------------------------------- | --------------------------------------------------- |
+| version               | String          |                                          | Java version to install                             |
+| variant               | String          | `openj9`                                 | Install falvour                                     | `hotspot` `openj9` `openj9-large-heap` |
```

Comment:
What's the right english choice for `flavour` vs. `flavor`?

Reply:
The English would tell you that `flavour` is the correct English choice :+1: